libbpf-tools/tcppktlat.c

```diff
@@ -0,0 +1,258 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2023 Wenbo Zhang
+#include <argp.h>
+#include <signal.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <time.h>
+#include <arpa/inet.h>
+#include <sys/socket.h>
+#include <sys/resource.h>
+
+#include <bpf/bpf.h>
+#include <bpf/libbpf.h>
+#include "tcppktlat.h"
+#include "tcppktlat.skel.h"
+#include "compat.h"
+#include "trace_helpers.h"
+
+static struct env {
+	pid_t pid;
+	pid_t tid;
+	__u64 min_us;
+	__u16 lport;
+	__u16 rport;
+	bool timestamp;
+	bool verbose;
+} env = {};
+
+static volatile sig_atomic_t exiting = 0;
+static int column_width = 15;
```

Comment:
Hi, Thank you for your great work. The width of the column is defined here, which is 15. There is no problem, but when I personally implement it, I would like to add an option, such as the -w option added by Hengqi Chen in tcplife.c

Reply:
Addressed, thanks!